src/Ombi/ClientApp/src/app/my-nav/my-nav.component.scss

```
@@ -1,260 +1,350 @@
-        background: $ombi-active;
+
+.mat-drawer-content {
+    position: relative;
+    overflow: hidden;
+    z-index: 1;
+    display: block;
+    height: 100%;
+}
+
+// ─── Overrides ───────────────────────────────────────────
+::ng-deep .mat-toolbar-row, .mat-toolbar-single-row {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)